make/Makefile.build.mk

```diff
@@ -86,7 +86,7 @@ test-e2e:
 ## run: Run kiali binary
```

Comment:
Delete lines 86-90 entirely - remove the "run" make target because it is obsolete and won't work even if you try it.